src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
-import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import './App.scss';
+
+export const App = () => {
+  const [query, setQuery] = useState('');
+
+  const filtered = moviesFromServer.filter(movie => {
+    const queryWords = query.trim().toLowerCase().split(/\s+/);
+
+    return queryWords.every(word => movie.title.toLowerCase().includes(word)
+    || movie.description.toLowerCase().includes(word));
+  });
```

Comment:
I'm sure you can find a simpler way to do it. I don't think`every` is the right 'tool' here - [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) ;)
It could be done using filter (just as you did) and a couple of includes (lowercasing and trimming as well)